ecommerce_shop_project/shop/models.py

```diff
@@ -77,6 +77,9 @@ class ProductImage(models.Model):
     product = models.ForeignKey(Product, related_name='images', on_delete=models.CASCADE)
     image = models.ImageField(upload_to="images/", default="")
 
+    def __str__(self):
+        return "image for {}".format(self.product)
```

Comment:
you can also add here the image filename so if a product has two (or more) images you will be able to see which one is this. Now for every image for one product you will see the same - just product name

Reply:
these names doesn't look very pretty. And I display the thumbnail of the image so you can't make a mistake.